ReShade/Shaders/Otis/MouseOverlay.fx

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////
+// This effects simply shows the mouse coordinates on the screen. It can be used in combination
+// of effects which use the mousecoordinates supplied by ReShade so you can better see what
+// you're doing.
+///////////////////////////////////////////////////////////////////
+
+#include EFFECT_CONFIG(Otis)
+#include "Common.fx"
+
+#if USE_MOUSEOVERLAY
+
+#pragma message "MouseOverlay by Otis\n"
+
+namespace Otis
+{
+	float4 PS_MouseCoordOverlay(float4 vpos:SV_Position, float2 texcoord: TEXCOORD) :SV_Target
+	{
+		float2 focusPoint = ReShade::MouseCoords * ReShade::PixelSize;
+		if(abs(focusPoint.x - texcoord.x) < MOL_CursorSize && abs(focusPoint.y - texcoord.y) < MOL_CursorSize)
```

Comment:
It will be faster if the first condition IS true, because in that case it has to evaluate two expressions. This way it only ever evaluates a single one, fully utilizing the SIMD instructions of the hardware.